### PR TITLE
Make com_joomlaupdate info and download URL opening a new page

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue raised by Michael Melson on Facebook

### Summary of Changes

Make sure the download URL and info URL opens as a new page.

### Testing Instructions

click the update URL or the info url

### Expected result

new page opens as it is a remote site

### Actual result

the current page is closed and the new page is open in the same tab.

### Documentation Changes Required

none

### CODE REVIEW

**Comment by brianteeman** [comment]
The reason that we did not put ` rel="noopener noreferrer"` on these links is that we are assuming that joomla.org is safe and doesn't need this security prevention.

In addition if you insist on the links opening in a new window then you must add information that the link opens in a new window.

**Comment by zero-24** [comment]
> The reason that we did not put rel="noopener noreferrer" on these links is that we are assuming that joomla.org is safe and doesn't need this security prevention.

In some cases we might use external sites like github too and IIRC it does not hurt right?

> In addition if you insist on the links opening in a new window then you must add information that the link opens in a new window.

Any suggestion how? IIRC we have an css class for this? Or do you think we should do it different?

**Comment by brianteeman** [comment]
No doesnt hurt

try something like < span class = sr-only > opens in a new window </ span

**Comment by zero-24** [comment]
> try something like < span class = sr-only > opens in a new window </ span

Do we already have an example / string in core for this so we are reinventing the wheel here also does this need to go in staging as this is a PR against staging?

**Comment by Quy** [comment]
@zero-24 Maybe this? https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_search/views/searches/tmpl/default.php#L72

**Comment by zero-24** [comment]
Please check the latest commits @brianteeman  @Quy

File: administrator/components/com_joomlaupdate/views/default/tmpl/default_reinstall.php [hunk 1]
```diff
@@ -31,8 +31,10 @@
 					<?php echo JText::_('COM_JOOMLAUPDATE_VIEW_DEFAULT_PACKAGE_REINSTALL'); ?>
 				</td>
 				<td>
-					<a href="<?php echo $this->updateInfo['object']->downloadurl->_data; ?>">
+					<a href="<?php echo $this->updateInfo['object']->downloadurl->_data; ?>" target="_blank" rel="noopener noreferrer">
 						<?php echo $this->updateInfo['object']->downloadurl->_data; ?>
+						<span class="icon-out-2" aria-hidden="true"></span>
+						<span class="sr-only"><?php echo JText::_('JBROWSERTARGET_NEW'); ?></span>
```
Comment (Quy) [inline administrator/components/com_joomlaupdate/views/default/tmpl/default_reinstall.php]: Please use `element-invisible` as `sr-only` is not available in the back end.

Comment (brianteeman) [reply administrator/components/com_joomlaupdate/views/default/tmpl/default_reinstall.php]: Sorry is sr-only j4 only
 I haven't used j3 for a yrar

Comment (zero-24) [reply administrator/components/com_joomlaupdate/views/default/tmpl/default_reinstall.php]: Fixed thanks :)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on ecc9606976af0b5230c521cf3e316bc959540084<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25848">issues.joomla.org/tracker/joomla-cms/25848</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on ecc9606976af0b5230c521cf3e316bc959540084<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25848">issues.joomla.org/tracker/joomla-cms/25848</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by HLeithner** [comment]
Thank you for making Joomla ux better.

**Pull request merged**